plugins/woocommerce-admin/client/products/product-page.tsx

```diff
@@ -8,11 +8,11 @@ import {
 	productApiFetchMiddleware,
 	TRACKS_SOURCE,
 	__experimentalProductMVPCESFooter as FeedbackBar,
-	__experimentalProductMVPFeedbackModalContainer as ProductMVPFeedbackModalContainer,
 	__experimentalEditorLoadingContext as EditorLoadingContext,
 } from '@woocommerce/product-editor';
+import { Spinner } from '@woocommerce/components';
 import { recordEvent } from '@woocommerce/tracks';
-import { useContext, useEffect } from '@wordpress/element';
+import React, { lazy, Suspense, useContext, useEffect } from 'react';
```

Comment:
We should try to use core API when possible. If I'm not wrong, all these imports are available in the `element` package.